src/stdpar/plugin-PixelVertexFinding/PixelVertexSoAFromCUDA.cc

```diff
@@ -1,48 +1,29 @@
-#include <cuda_runtime.h>
-
-#include "CUDACore/Product.h"
 #include "CUDADataFormats/ZVertex.h"
 #include "Framework/EventSetup.h"
 #include "Framework/Event.h"
 #include "Framework/PluginFactory.h"
 #include "Framework/EDProducer.h"
 #include "Framework/RunningAverage.h"
-#include "CUDACore/ScopedContext.h"
 
-class PixelVertexSoAFromCUDA : public edm::EDProducerExternalWork {
+class PixelVertexSoAFromCUDA : public edm::EDProducer {
```

Comment:
In principle this module is also useless now (well, actually earlier but I realized it only now).